### DISCUSSION TITLE
RustDesk keep Closing when not connected to relay server

### DISCUSSION THREAD

**Discussion by fdciabdul** [rustdesk/rustdesk#6941] [General]
### Bug Description

https://github.com/rustdesk/rustdesk/assets/31664438/b6035e2a-9a21-4d77-be09-949fc4506d30

After my self-host Rustdesk server has been shutdown and inactive , the rustdesk keep closing

the problem happen since i click this

![untitled](https://github.com/rustdesk/rustdesk/assets/31664438/d1b95516-4dbf-4102-bb0c-2dd0e3e08895)

### How to Reproduce

- Open Rust Desk 
- Set own server
- Server inactive 
- Click text on bottom app
- Force close , even rust desk has been uninstall and download nightly build

### Expected Behavior

I want change the server to different server

### Operating system(s) on local side and remote side

Windows 11

### RustDesk Version(s) on local side and remote side

Stable build and new nightly build

### Screenshots

![untitled](https://github.com/rustdesk/rustdesk/assets/31664438/d1b95516-4dbf-4102-bb0c-2dd0e3e08895)

**Comment by xlionjuan** [comment]
It mean client can't communicate to ID server(hbbs) not Relay (hbbr)

**Comment by PotatoYummy** [comment]
+1

**Comment by fdciabdul** [comment]
fixed , i delete Rustdesk folder in Program Files

**Comment by 109-999** [comment]
I was getting your message.do in touch.

**Comment by krupennikov** [comment]
what is the solution to this bug? I have the same problem.
